apps/namada-interface/src/App/types.ts

```diff
@@ -83,8 +83,8 @@ export const locationToStakingAndGovernanceSubRoute = (
 };
 
 export enum LocalStorageKeys {
-  MasterSeed = "com.anoma.network:seed",
-  Session = "com.anoma.network:session",
-  Persist = "com.anoma.network:state",
-  ColorMode = "com.anoma.color-mode",
+  MasterSeed = "com.namada.network:seed",
+  Session = "com.namada.network:session",
+  Persist = "com.namada.network:state",
```

Comment:
Not a part of your PR, but somehow these types have made it back in (I thought I had removed the unused ones!) - I'll keep an eye out for it in my next PR and clean that up, as at least the `MasterSeed` and `Session` are unused